deps/chakrashim/src/v8handlescope.cc

```diff
@@ -42,8 +42,7 @@ HandleScope::~HandleScope() {
   while (currRecord != nullptr) {
     AddRefRecord * nextRecord = currRecord->_next;
     // Don't crash even if we fail to release the scope
-    JsErrorCode errorCode = JsRelease(currRecord->_ref, nullptr);
-    CHAKRA_ASSERT(errorCode == JsNoError);
+    JsRelease(currRecord->_ref, nullptr);
```

Comment:
No need to. This only removes a compiler warning. Original code help debugging a little bit. Alternatively can change them all to "CHAKRA_ASSERT(...); UNUSED(errorCode);". Just felt the assertions are not that important in most of the cases.